qemu/qemu.go

```diff
@@ -995,28 +963,6 @@ func (vsock VSOCKDevice) Valid() bool {
 	return true
 }
 
-// QemuParams returns the qemu parameters built out of the VSOCK device.
-func (vsock VSOCKDevice) QemuParams(config *Config) []string {
```

Comment:
I don't understand this change , did you just move the function? if this is the case could you please revert it?

Reply:
@devimc Z does not support vhost-user-blk device so this test was oved into qemu_generic_test.go and qemu_s390x_test.go. With suitable changes applied.